organization/remuneration-policy.md

```diff
@@ -0,0 +1,38 @@
+---
+type: resource
+---
+
+# Remuneration policy
+
+This explains the Foundation for Public Code's approach to remuneration.
+
+## Staff, contractors and interns
+
+Foundation staff salaries are competitive as well as in line with the social mission of the organization.
+
+> Interns may or may not be compensated, depending on whether they're part of a work placement scheme or have an external stipend. We believe strongly that everyone who works for us should receive a living wage. (FACTCHECK?)
+
+The Foundation will reimburse reasonable expenses for staff, contractors and interns.
+
+## Board of directors
+
+> Translate this info: Bestuurders krijgen alleen een vergoeding voor gemaakte onkosten. Ze ontvangen geen vacatiegeld. Gemaakte onkosten zijn kosten die bestuursleden redelijkerwijs hebben door hun rol als bestuurder bij de Foundation.
+
+Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
+
+> Do directors get different expenses reimbursed than non-directors?
```

Comment:
@MirjamvT Directors' expenses were worded differently from non-directors; is this significant?